swan-lake/development-tutorials/ballerina-central/publish-packages-to-ballerina-central.md

```diff
@@ -66,7 +66,14 @@ You can publish a Ballerina archive to the <a href="https://central.ballerina.io
 
 2. Navigate to the <a href="https://central.ballerina.io/dashboard?tab=token" target="_blank">Dashboard</a> and acquire an access token.
 
-3. Download and place the `Settings.toml` file in your home repository (`<USER_HOME>/.ballerina/`). If you already have a `Settings.toml` file configured in your home repository, follow the other option and copy the access token into the `Settings.toml`. 
+3. Download and place the `Settings.toml` file in your home repository (`<USER_HOME>/.ballerina/`). If you already have a `Settings.toml` file configured in your home repository, follow the other option and copy the access token into the `Settings.toml` as follows.
+
+    ```Toml
+    [central]
+    accesstoken="<token>"
+    ```
+
+> **Note:** These generated tokens are valid only for one year from the generation date. Upon the expiration of the token, a new token should be generated, and `Settings.toml` should be updated with it. Otherwise, you may get an error related to unauthorized access.
```

Comment:
```suggestion
> **Note:** The tokens generated are valid for one year from the generation date. Upon expiration, you must generate a new token and update the `Settings.toml` file to avoid unauthorized access errors.
```